packages/cli/src/PublicApi/v1/handlers/workflows/workflows.handler.ts

```diff
@@ -106,17 +106,24 @@ export = {
 
 			if (['owner', 'admin'].includes(req.user.globalRole.name)) {
 				if (tags) {
-					const workflowIds = await getWorkflowIdsViaTags(parseTagNames(tags));
+					const workflowIds = await Container.get(TagRepository).getWorkflowIdsViaTags(
```

Comment:
Once we are done moving all DB stuff into repositories, we should start moving all logic into services, and make sure that the public api and the rest api reuse as much code as possible. we have currently too much duplication in the public api.

Reply:
our code used to be a lot less re-usable because of how it was structured in the past.